proto/umee/leverage/v1/events.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+package umee.leverage.v1;
+
+import "google/api/annotations.proto";
+import "umee/leverage/v1/leverage.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/leverage/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+message EventSupply {
+  // Liquidity supplier bech32 address.
+  string supplier = 1;
+  // Liquidity provided to the module.
+  cosmos.base.v1beta1.Coin supplied = 2;
+  // uAsset received by the supplier in exchange for the
+  // provided liquidity.
+  cosmos.base.v1beta1.Coin uasset = 3;
+}
+
+message EventWithdraw {
+  // Liquidity supplier bech32 address.
+  string supplier = 1;
+  // uAsset sent to the module in exchange for the underlying asset.
+  cosmos.base.v1beta1.Coin redeemed = 2;
+  // Liquidity received by the supplier.
+  cosmos.base.v1beta1.Coin asset = 3;
```

Comment:
renamed `received` -> `assert` (to be more inline with EventSupply)

Reply:
`received: 100uumee` is more informative than `asset: 100uumee`

Reply:
this is old version. I'm using asset / uToken to better follow the correct names we use in our docs.